tests/packagedcode/test_npm.py

```diff
@@ -543,6 +543,24 @@ def test_npm_scan_with_private_package_json_and_yarn_lock(self):
             expected_file, result_file, remove_uuid=True, regen=REGEN_TEST_FIXTURES
         )
 
+    def test_parse_npm_package_json_with_overrides(self):
```

Comment:
Tests will look like this: https://github.com/aboutcode-org/scancode-toolkit/blob/56aa1e7209120b56b79012f36762d1920aa9c7aa/tests/packagedcode/test_npm.py#L191